File: Back/src/ProEventos.Domain/Evento.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ProEventos.Domain
{
    public class Evento
    {
        public int Id { get; set; }
        public string Local { get; set; }
        public DateTime? DataEvento { get; set; }
        public string Tema { get; set; }
        public int QtdPessoas { get; set; }
        public string Lot { get; set; }
        public string ImageURL { get; set; }
        public string Telefone { get; set; }
        public string Email { get; set; }
        public IEnumerable<Lot> Lots { get; set; }
        public IEnumerable<RedeSocial> RedeSociais { get; set; }
        public IEnumerable<PalestrantEvento> PalestrantEventos { get; set; }
    }
}

File: Front/ProEventos-App/src/app/models/Evento.ts
import { Lot } from './Lot';
import { Palestrant } from './Palestrant';
import { RedeSocial } from './RedeSocial';

export interface Evento {
  id: number;
  local: string;
  dataEvento?: Date;
  tema: string;
  qtdPessoas: number;
  imageURL: string;
  telefone: string;
  email: string;
  lots: Lot[];
  redeSociais: RedeSocial[];
  palestrantEventos: Palestrant[];
}
